sklearn/datasets/svmlight_format.py

```diff
@@ -98,6 +103,15 @@ def load_svmlight_file(f, n_features=None, dtype=np.float64,
         Data type of dataset to be loaded. This will be the data type of the
         output numpy arrays ``X`` and ``y``.
 
+    offset : integer, optional (default is 0)
```

Comment:
Nitpick: I don't know the official way of mentioning the default, but it looks like it uses:
```
..., default default_value
```
rather than
```
... (default is default_value)
```